Euphorolog.Database/ModelConfigurations/UsersConfiguration.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Euphorolog.Database.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Euphorolog.Database.ModelConfigurations
+{
+    public class UsersConfiguration : IEntityTypeConfiguration<Users>
+    {
+        public void Configure(EntityTypeBuilder<Users> builder)
+        {
+            //userName
+            builder
+                .HasKey("userName");
+            builder
+                .Property(u => u.userName)
+                .IsRequired()
+                .HasMaxLength(55);
```

Comment:
If the primary key is auto generated by database, no need to define a Max Length. But if you do that yourself, then it is fine. Although it is unconventional for most cases to define it yourself.